apps/backend/src/adoption/repositories/adoption-repository.interface.ts

```diff
@@ -0,0 +1,23 @@
+import { AdoptionType, UserType } from '@animavita/models';
+import { AdoptionResponse } from '../adoption.interface';
+import { Location } from '../../user/repositories/mongodb/user.interface';
+
+export type IAdoption = AdoptionType & {
+  user: string;
```

Comment:
What does `user` means in this case? Is it an ID? Or the user name? 
Can we rename it to something else?

Reply:
Yeah, this is supposed to be the `userId`, good catch!